content/realtime/channels/channel-parameters/deltas.textile

```diff
@@ -72,57 +84,30 @@ TBD
 TBD
```

Comment:
@QuintinWillison 2 TBD examples remaining

Reply:
Thanks. These are on my list to be worked on today or, worst case, early next week.

Reply:
@MarkWoulfeAbly ...

Cocoa is [here](https://github.com/ably/hello-deltas-cocoa/blob/68dfa96ad5f539c4c9158b12c0ffb5837dd05f10/HelloDeltas/main.swift#L43) (though Swift, not Objective-C), thanks to @ricardopereira.

I'm looking at .NET today.

Reply:
will resolve with tom and quintin in a code review meeting.